scripts/code_generator.rb

```diff
@@ -54,6 +54,10 @@ def append_to_index(root_class_name, index_file)
       end
       prev_c = c
     end
+    if root_class_name.start_with? 'Openid'
+      root_class_name.sub!('Openid', 'OpenID')
```

Comment:
@filmaj I changed my mind. The generator script replaces `Openid` with `OpenID` now.

Reply:
Nice!